packages/block-library/src/group/edit.js

```diff
@@ -43,6 +66,21 @@ function GroupEdit( {
 						},
 					] }
 				/>
+
+				<PanelBody title={ __( 'Spacing' ) }>
+					<ResponsiveBlockControl
+						legend="Padding"
+						property="padding"
+						responsiveControlsActive={ true }
+						renderDefaultControl={ ( label ) => (
+							<SelectControl
+								label={ <ResponsiveBlockControlLabel property="Padding" device={ label } /> }
```

Comment:
Can't the label passed as prop already include the component ResponsiveBlockControlLabel rendered, so we don't need to expose ResponsiveBlockControlLabel at all?

Reply:
I've been wrestling with this one.

I've just realised I could pass the `<ResponsiveBlockControlLabel />` component as an argument to the `renderDefaultControl` render prop. This allow the render prop function to consume the label as a component and utilise that however it needs.

In the example case of a `<SelectControl />` we simply pass the label component as the `label` prop and everything "just works". A more complex `renderDefaultControl` use case could use the component directly in a more custom manner.

Thanks for prompting me to reconsider my approach.

Note I've left `ResponsiveBlockControlLabel` exposed as the moment but I guess we could remove the `export`?

Reply:
This has now all been resolved by handling the label internally and removing the additional `export`.